ruler/app.py

```diff
@@ -141,7 +141,8 @@ def create(event, context):
             "AuthenticationRequestExtraParams": {
               "claims": "{\"id_token\":{\"userid\":{\"essential\":true}},\"userinfo\":{\"userid\":{\"essential\":true}}}"
             },
-            "OnUnauthenticatedRequest": "authenticate"
+            "OnUnauthenticatedRequest": "authenticate",
+            "SessionTimeout": 3600
```

Comment:
most if not all of the values for this rule are [passed in from cloudformation parameters](https://github.com/Sage-Bionetworks-IT/cfn-cr-alb-rule?tab=readme-ov-file#install-lambda-into-aws).  it would be nice if this  could be passed in as well.